commands/details.js

```diff
@@ -46,9 +50,23 @@ async function details (argv, arg1, arg2, arg3) {
     return
   }
 
+  const tree = await universalModuleTree(dir)
+  const list = universalModuleTree.flatten(tree)
+  let requirePaths = []
+  for (const pkg of list) {
+    if (pkg.name === name && pkg.version === version) {
+      requirePaths = pkg.paths
+      break
+    }
+  }
+
   /* NCM-Cli Header */
   L()
-  L(header(`${name} @ ${version}`))
+  if (requirePaths.length > 0) {
+    L(header(`${name} @ ${version} (within ${path.basename(dir)})`))
+  } else {
+    L(header(`${name} @ ${version}`))
+  }
```

Comment:
for reviewing this it would be handy if you attached screenshots of the command output in the PR